test/inlining/bug_gh6303.js

```diff
@@ -0,0 +1,14 @@
+// -mic:1 -off:simplejit
```

Comment:
Are these flags actually required for the repro? Because they aren't in rlexe.xml.

Reply:
This is leftover from my debugging workflow - I'll remove it!